python/pyspark/errors/__init__.py

```diff
@@ -59,4 +61,5 @@
     "PySparkTypeError",
     "PySparkValueError",
     "PySparkAttributeError",
+    "CapturedException",
```

Comment:
Can you remove this here but add it into `python/pyspark/sql/utils.py` alone? Adding `__all__` implies that it's an API

Reply:
Updated